src/main/scala/mix/kafka/monitor/MetricsCacheProvider.scala

```diff
@@ -0,0 +1,57 @@
+package mix.kafka.monitor
+
+import java.time.Duration
+import java.util.concurrent.TimeUnit
+
+import com.codahale.metrics.MetricRegistry
+import kafka.utils.Logging
+import mix.kafka.monitor.LibratoOffsetReporter.getMetricName
+
+case class GroupTopic(group: String, topic: String)
+
+class MetricsCacheProvider(metricRegistry: MetricRegistry, cacheExpiration: Duration) extends Logging {
+
+  import java.lang.{Integer => JInt, Long => JLong}
+
+  logger.info(s"starting MetricsCacheProvider with cacheExpiration = $cacheExpiration")
+
+  def existingCacheEntries(keys: Seq[GroupTopic]): Map[GroupTopic, Map[JInt, JLong]] = {
+    import scala.collection.JavaConverters._
```

Comment:
Since ` import scala.collection.JavaConverters._` is used in multiple methods, can we move it up ?